src/hooks/useConnectCurrentVenue.ts

```diff
@@ -1,8 +1,16 @@
 import { useParams } from "react-router-dom";
 import { useFirestoreConnect } from "react-redux-firebase";
+
+import { currentVenueSelector } from "utils/selectors";
 import getQueryParameters from "utils/getQueryParameters";
 
-const useConnectCurrentVenue = () => {
+import { useSelector } from "./useSelector";
+
+/**
+ * @deprecated use useConnectCurrentVenueNG instead
+ * @see useConnectCurrentVenueNG
+ */
+export const useConnectCurrentVenue = () => {
```

Comment:
Given the issues we saw introduced in https://github.com/sparkletown/sparkle/commit/f7798176b0d77c28653247e15ed8466815cf2c10 (within https://github.com/sparkletown/sparkle/pull/721), we need to be careful about switching things across to use this, so it may not be so much "don't use this, use the other" at the moment.